recipes/systemc-cci/1.0.0/test_package/example.cpp

```diff
@@ -0,0 +1,6 @@
+#include <cci_configuration>
+
+int sc_main(int argc, char *argv[])
```

Comment:
I don't think this is testing systemc-cci but systemc.

Reply:
There is not much to test for CCI without implementing a full systemc model.

Further 'sc_main' is required because of linking againsts systemc

Reply:
Would a model of a not gate suffice? It doesn't have to be complex.

Reply:
Just created a global CCI broker should be more then enough to check that CCI lib is build correctly